gui/package.json

```diff
@@ -12,20 +12,25 @@
     "css-loader": "^0.9.1",
     "file-loader": "^0.8.1",
     "url-loader": "^0.5.6",
-    "babel": "^5.8.23",
-    "babel-core": "^5.8.23",
-    "babel-eslint": "^4.1.1",
-    "babel-loader": "^5.3.2",
-    "babel-plugin-react-transform": "^1.1.1",
+    "babel": "^6.1.0",
+    "babel-core": "^6.1.0",
+    "babel-loader": "^6.1.0",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-plugin-transform-object-rest-spread": "^6.1.0",
+    "babel-plugin-transform-runtime": "^6.1.0",
+    "babel-preset-es2015": "^6.1.0",
+    "babel-preset-react": "^6.1.0",
+    "babel-preset-stage-0": "^6.1.0",
+    "babel-register": "^6.1.0",
```

Comment:
@arbonap @minifast these dependencies weren't needed before Jasmine unit testing was introduced, yes?  To the best of my knowledge, these dependencies were added to "appease Travis" and to get Jasmine working properly.